mne/commands/mne_setup_source_space.py

```diff
@@ -0,0 +1,144 @@
+"""Set up bilateral hemisphere surface-based source space with subsampling.
+
+Examples
+--------
+.. code-block:: console
+
+    $ mne setup_source_space --subject sample
+
+
+ .. note : Only one of --ico, --oct or --spacing options can be set at the same
+           time. Default to spacing = 7mm.
+
+"""
+
+import sys
+
+import mne
+
+
+def run():
+    """Run command."""
+    from mne.commands.utils import get_optparser
+
+    parser = get_optparser(__file__)
+
+    parser.add_option('-s', '--subject',
+                      dest='subject',
+                      help='Subject name (required)',
+                      default=None)
+    parser.add_option('--src', dest='fname',
+                      help='Output file name. Use a name <dir>/<name>-src.fif',
+                      metavar='FILE', default=None)
+    parser.add_option('--morph',
+                      dest='subject_to',
+                      help='morph the source space to this subject',
+                      default=None)
+    parser.add_option('--surf',
+                      dest='surface',
+                      help='The surface to use. (default to white)',
+                      default='white',
+                      type='string')
+    parser.add_option('--spacing',
+                      dest='spacing',
+                      help='Specifies the approximate grid spacing of the '
+                           'source space in mm. (default to 7mm)',
+                      default=None,
+                      type='int')
+    parser.add_option('--ico',
+                      dest='ico',
+                      help='use the recursively subdivided icosahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('--oct',
+                      dest='oct',
+                      help='use the recursively subdivided octahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('-d', '--subjects-dir',
+                      dest='subjects_dir',
+                      help='Subjects directory',
+                      default=None)
+    parser.add_option('--cps',
+                      dest='cps',
+                      help='Add patch information to source space.',
+                      default=True)
+    parser.add_option('-n', '--n-jobs',
+                      dest='n_jobs',
+                      help='The number of jobs to run in parallel '
+                            '(default 1). Requires the joblib package. '
+                            'Will use at most 2 jobs'
+                            '(one for each hemisphere).',
+                      default=1,
+                      type='int')
+    parser.add_option('--verbose',
+                      dest='verbose',
+                      help='Turn on verbose mode.',
+                      default=None)
```

Comment:
`mne setup_source_space -s sample --oct=6 --verbose`

does not work.

Reply:
maybe it's common to more commands but here the help message is not clear.

Reply:
Does somethings like "if used, set verbose mode to 'INFO' (default to 'WARNING'). " is ok ?

Reply:
you need `action=store_true` and then it should work, see how we do it in other commands with `verbose` arg. At least I've been trying to make it so that all of them do it the same way that it's done in `browse_raw`:
```
    parser.add_option("--verbose", dest='verbose',
                      help="Enable verbose mode.", default=None,
                      action="store_true")
```

Reply:
> 
> 
> maybe it's common to more commands but here the help message is not clear.

Any hint for a clearer solution ?

Reply:
We can add a private `_add_verbose_switch` that unifies these. I can do this quickly today, get it into `master`, and you can use it here.

Reply:
@vferat feel free to rebase now that #6786 is in and use `_add_verbose_flag`

Reply:
Thx @larsoner , maybe some other flags such as --overwrite and --n-cpu that are shared by several command lines could be unified as well in a near futur.